lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -212,6 +207,19 @@ div.course-wrapper {
           }
         }
       }
+
+      .vert > .xblock-student_view {
+        @extend .clearfix;
+        border-bottom: 1px solid #ddd;
+        margin-bottom: ($baseline*0.75);
+        padding: 0 0 15px;
+      }
+
+      .vert > .xmodule_HiddenModule {
+          display: none;
+          border-bottom: 0px;
+          margin-bottom: 0px;
+      }
```

Comment:
That all makes sense, though one thing I would suggest is to move toward having a state-based class for this kind of styling change. 

For example, is-deprecated might be an additional class added to the xblock class itself (the highest level within a given deprecated xblock. When is-deprecated (or is-hidden) is added at this level, we have some flexibility about say, nest styles within this case to have  .xmodule_HiddenModule be display:none, etc.

Let me know if that makes sense or if you have any other questions. I've included a reference link below as well with more details about the class-based styling convention. 

(Relevant reference: https://smacss.com/book/type-state)
(UX Pattern Library docs - we also talk about state-based classes and SMACSS here: https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS) 

Thanks for tagging me here!